.github/workflows/ci.yaml

```
@@ -49,11 +41,49 @@ jobs:
+    - name: Verify committed capnp code is up to date
+      run: |
+          git diff --exit-code -- hyperfuel-net-types/src/__generated__ \
+            || (echo "::error::Committed capnp code is out of date. Run 'make -C hyperfuel-net-types generate_capnp_types' and commit the result." && exit 1)
+
+  # Verify the publishable crates package and verify-build correctly. This builds
+  # against the versions on crates.io, which still need the capnp compiler until
+  # a pre-built hyperfuel-net-types is published, so keep capnproto installed here.
+  package:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)